### PR TITLE
fix: Wait for SSP operator to be ready after TLS policy tests

### PR DESCRIPTION
**Pull request by akrejcir**
**What this PR does / why we need it**:
The TLS policy tests cause the operator to enter crash loop. This commit adds code that waits until it is ready.

**Release note**:
```release-note
None
```

### CODE REVIEW

File: tests/crypto_policy_test.go [hunk 1]
```diff
@@ -103,6 +106,17 @@ var _ = Describe("Crypto Policy", func() {
 
 	AfterEach(func() {
 		strategy.RevertToOriginalSspCr()
+
+		// The SSP operator may enter crash loop backoff state, so we need to wait until it is running
```
[the remark this answers is not in the document]
Comment (akrejcir) [reply tests/crypto_policy_test.go]: Done

File: tests/crypto_policy_test.go [hunk 2]
```diff
@@ -103,6 +106,19 @@ var _ = Describe("Crypto Policy", func() {
 
 	AfterEach(func() {
 		strategy.RevertToOriginalSspCr()
+
+		// Because of bug[1], the SSP operator will move to CrashLoopBackOff state,
```
Comment (lyarwood) [inline tests/crypto_policy_test.go:110]: supernit - I forget what the rules are for FIXMEs in this codebase but this strikes me as something we could mark as FIXME once that bug is resolved?

Comment (akrejcir) [reply tests/crypto_policy_test.go:110]: There are no rules. Do you have a suggestion for what is a good way to mark as FIXME? Or simply prefix the comment with `FIXME:` ?

**Comment by dominikholler** [comment]
@akrejcir Please do not mention the bug in the commit message or in the PR description, because this would make the automation assume that this PR/commit would contain already the fix the the bug.

**Comment by 0xFelix** [comment]
I think we have to reconsider how the automation works. If we exactly know the issue (speaking of a known bug), we should be able to add a direct reference to it. Does the automation really pick up links in the code too? Or just in the description of a commit?

**Review by 0xFelix — APPROVED** [review]
/approve

**Pull request merged**